### DISCUSSION TITLE
why w_s,f follow gamma distribution

### DISCUSSION THREAD

**Discussion by caiquanyou** [BayraktarLab/cell2location#261] [General]
### Problem

Icannot understand why w_s,f follow gamma distribution ？

**Comment by vitkl** [comment]
Hi @caiquanyou 

Gamma distribution is a prior for w_s,f because it is a distribution over positive real numbers. Does this answer your question? If not please describe exactly what you don't understand.

**Comment by vitkl** [comment]
I am assuming that this is resolved.
